robocopkaka/hospital
# chore(routes): protect routes

#### What does this PR do?
Protects routes

 #### Description of Task to be completed?
Prevent patients from seeing the link to see all doctors and vice versa

 #### How should this be manually tested?
* Log in/Sign up as a doctor
* You should only see a link for `Doctors` besides `Home` on your navbar
* Log in/Sign up as a patient
* You should only see a link for `Patients` besides `Home` on your navbar
 #### What are the relevant pivotal tracker stories?
[#160661993](https://www.pivotaltracker.com/story/show/160661993)
 #### Any background context you want to add?
N/A
